pd/src/consensus/worker.rs

```diff
@@ -96,11 +99,8 @@ impl Worker {
 
         // Now start building the genesis block:
         self.note_commitment_tree = NoteCommitmentTree::new(0);
-        let mut genesis_block = PendingBlock::new(
-            self.note_commitment_tree.clone(),
-            app_state.chain_params.epoch_duration,
-        );
-        genesis_block.set_height(0);
+        let mut genesis_block = PendingBlock::new(self.note_commitment_tree.clone());
```

Comment:
We don't need the epoch duration to construct a `PendingBlock` anymore, because it doesn't directly contain this information anymore. This was duplicated from the chain parameters, so I removed the duplication.